example/5-create_and_attach_disk.sh

```diff
@@ -89,7 +89,7 @@ SOCK="$BIN_DIR/$id.sock"
 
 sudo modprobe nbd
 touch $SOCK
-sudo $BIN_DIR/$NBD --device-mode endpoint --disk-id $id --access-mode rw \
+sudo LD_LIBRARY_PATH=$LD_LIBRARY_PATH $BIN_DIR/$NBD --device-mode endpoint --disk-id $id --access-mode rw \
```

Comment:
don't you need to set LD_LIBRARY_PATH previously to make this useful ...
probably something like
```
export LD_LIBRARY_PATH=$BIN_DIR
```

Reply:
LD_LIBRARY_PATH is set here in [example/README.md](https://github.com/ydb-platform/nbs/pull/94/files#diff-bc501e6e1e9420017b2d954e834fd8c3afb7daf9e2eb780c46c491099a7c2f0eR22)

There are some difficulties here;
* ```ya``` and ```ninja``` produce output in different folders
* $BIN_DIR does not contain libraries, but only binaries
* There is no the only one library folder with ```ya``` tool, instead the libraries are copied alongside with binraies

If you have an idea how to make it more clear please post it here

Reply:
we can also patch rpath in binaries instead of using the env variable
for example
```
apt install patchelf
NBSD=readlink -f ../cloud/blockstore/apps/server/nbsd
patchelf --set-rpath $(dirname $NBSD):'$ORIGIN' $NBSD
```

Reply:
I decided to leave the option with LD_LIBRARY_PATH, but put it in a separate file. It now looks clear for me. @budevg , could you check please